package.json

```diff
@@ -83,19 +83,19 @@
     "jest-runner-stylelint": "1.0.0",
     "jest-silent-reporter": "0.1.2",
     "jest-watch-typeahead": "0.3.1",
-    "lerna": "3.13.4",
+    "lerna": "3.14.1",
     "lerna-changelog": "0.8.2",
-    "lint-staged": "8.1.6",
-    "prettier": "1.17.0",
-    "puppeteer": "1.15.0",
+    "lint-staged": "8.1.7",
+    "prettier": "1.17.1",
+    "puppeteer": "1.16.0",
     "rcfile": "1.0.3",
     "read-pkg-up": "6.0.0",
-    "rollup": "1.11.3",
+    "rollup": "1.12.1",
     "rollup-plugin-babel": "4.3.2",
-    "rollup-plugin-commonjs": "9.3.4",
+    "rollup-plugin-commonjs": "10.0.0",
```

Comment:
I was looking at this, I think the "breaking" change is it requires rollup the new rollup, which you updated as well 👍

Reply:
Yes. That's how I understood it too when reading through their PRs.